phpcs.xml

```diff
@@ -342,17 +314,8 @@
   <rule ref="Squiz.WhiteSpace.LanguageConstructSpacing" />
   <rule ref="Squiz.WhiteSpace.SemicolonSpacing"/>
   <rule ref="Squiz.WhiteSpace.SuperfluousWhitespace"/>
-  <rule ref="Squiz.WhiteSpace.OperatorSpacing">
-    <properties>
-      <property name="ignoreNewlines" value="true"/>
-    </properties>
-  </rule>
 
   <!-- Zend sniffs -->
   <rule ref="Zend.Files.ClosingTag"/>
 
-  <!-- sniffs from https://github.com/acquia/coding-standards-php -->
-  <rule ref="SlevomatCodingStandard.Namespaces.AlphabeticallySortedUses" />
-  <rule ref="SlevomatCodingStandard.Variables.DisallowSuperGlobalVariable" />
```

Comment:
We can also keep this to discourage direct use of $_GET, $_POST, $_SERVER and $GLOBALS etc.